### ISSUE TITLE
Advertised fine-tuning hyper-parameters & results

### ISSUE THREAD

**Issue by saareliad** [google-research/vision_transformer#29]
Hi,
I am trying to restore your fine-tuning results on ViT-L-32 Imagenet (and larger model later)

I notice you publish two tensorboard.dev runs reaching ~81.7 and ~82 accuracies.
(final train loss is ~0.4 and ~0.8 ), the hyperparameters difference between not visible from tensorboard.dev.
Can you clarify the difference between the two runs?
Is there a reproduction script for these results with the code here, or does it include additional addons e.g Polyak avg?

(The reported accuracy in the paper was 80.99)

In my attempt to reproduce with same hyper parameters, I reached a lower training loss  (~0.16) after 20K steps used (lr 0.03 according to graph tensorboard.dev ). Unfortunately, my training doesn't match tensrobard.dev contours not final result.
The top accuracy I see at an early epoch is 80.98, then I observe slight overfitting.

**Comment by andsteing** [comment]
Hi Saar

Thanks for bringing this to my attention.

There was no difference between these two runs. Note that the `train_loss` is very noisy (at least in part due to data augmentation), if you smooth the values, you get a much smaller difference (0.88 vs 0.85). The variance between individual runs could be relatively high though, I only ran two models cited in the table for reproduction in the cloud setup so far.

There was a bug that made results before 11/3/20 (i.e. including the [ViT-L_32/imagenet2012 runs]) effectively use `dropout=0` - I added a note to the `README` in [f952d6]. It's surprising though that you get more overfitting compared to the [ViT-L_32/imagenet2012 runs] that do not use dropout.

Other than that, the configuration should be identical (the only changes from the default flags was adjusting `lr=0.01` for `ViT-B_32/cifar10`). I'm now rerunning the `ViT-L_32/imagenet2012` 2x more with `config.transformer.dropout_rate=0.1` to see if I get consistently lower final accuracy.

Best, Andreas

[f952d6]: https://github.com/google-research/vision_transformer/commit/f952d612e1b55d1099b2fbf87fc04218d5c4fe18
[ViT-L_32/imagenet2012 runs]: https://tensorboard.dev/experiment/CVAxQLz1Rwycg9WgqNG80Q/#scalars&_smoothingWeight=0&regexInput=imagenet21k/ViT-L_32/imagenet2012/

**Comment by andsteing** [comment]
Hi Saar

I trained the `ViT-L_32` model twice more and got final accuracies of 81.20% and 81.22% (0.99 smoothed final train loss 0.68 and 0.78) :

https://tensorboard.dev/experiment/TYOsMas0SaSBtmruMdYzQA

So dropout does not seem to make a big difference (both models were trained from `HEAD`). Note that in both runs there is no overfitting, but the test accuracy increases until the end of the training.

So I think that if you repeat the finetuning a couple of times you should well achieve final accuracies in the 81.2% range.

Best, Andreas
